src/Components/Grants/GrantShowOverview.jsx

```diff
@@ -1,29 +1,140 @@
-/* eslint-disable no-unused-vars */
-import React, { useState } from "react";
+import React, { useEffect } from "react";
 import { useQuery, useMutation } from "react-query";
 import { useParams } from "react-router-dom";
+import Button from "../design/Button/Button";
 import Container from "../design/Container/Container";
-import SectionListItem from "../Sections/SectionListItem";
+import Hero from "../design/Hero/Hero";
+import { useCurrentOrganization } from "../../Contexts/currentOrganizationContext";
+import * as GrantsService from "../../Services/Organizations/GrantsService";
+import * as SectionsService from "../../Services/Organizations/Grants/SectionsService";
+import countSectionWords from "../../Helpers/countSectionWords";
+import { SortableItem } from "./SortableItem";
+import { Item } from "./Item";
+import CurrentOrganizationLink from "../Helpers/CurrentOrganizationLink";
+import {
+  SortableContext,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { DragOverlay } from "@dnd-kit/core";
+
 import "./GrantShowOverview.css";
-import SortableElement from "../Elements/SortableElement";
 
-export default function GrantShowOverview({ grant }) {
+function countTotalSectionsWords(sections = []) {
+  return sections?.reduce(
+    (total, section) => total + countSectionWords(section),
+    0
+  );
+}
+
+export default function GrantShowOverview(props) {
+  const { currentOrganization, organizationClient } = useCurrentOrganization();
+  const { grantId } = useParams();
+
+  const {
+    data: grant,
+    isError,
+    isLoading,
+    error,
+  } = useQuery("getGrant", () =>
+    GrantsService.getGrant(organizationClient, grantId)
+  );
+  const totalWordCount = countTotalSectionsWords(grant?.sections);
+
+  const grantSectionsReorder = () => {
+    const sectionsToReorder = [];
+    props.sortableSections.forEach((newSection, index) => {
+      const checkSection = grant.sections[index];
+      if (newSection.sortOrder !== checkSection.sortOrder) {
+        sectionsToReorder.push({
+          id: newSection.id,
+          sort_order: index,
+        });
+      }
+    });
```

Comment:
Wyt of using a filter+map instead of forEach+push? Something like this:
```suggestion
    const sectionsToReorder = props.sortableSections
      .filter((newSection, index) => {
        const checkSection = grant.sections[index];
        return newSection.sortOrder !== checkSection.sortOrder;
      })
      .map((section, index) => ({ id: section.id, sort_order: index }));
```

Reply:
Also, since this is called when the button is clicked, maybe we rename this to something like `handleClickReorderSections` or `handleReorderSections`